webui/src/pages/auth/groups/group/members.jsx

```diff
@@ -26,15 +26,41 @@ const GroupMemberList = ({ groupId, after, onPaginate }) => {
     const [refresh, setRefresh] = useState(false);
     const [showAddModal, setShowAddModal] = useState(false);
     const [attachError, setAttachError] = useState(null);
-
+    const [allUsers, setAllUsers] = useState([]);
     const {results, loading, error, nextPage} = useAPIWithPagination(() => {
         return auth.listGroupMembers(groupId, after);
     }, [groupId, after, refresh]);
-
     useEffect(() => {
         setAttachError(null);
     }, [refresh]);
 
+    const setAllUsersFromLakeFS = async () => {
+        if (allUsers.length > 0) {
+            return allUsers
+        }
+        after = ""
+        let hasMore = true
+        let usersList = []
+        try {
+            do {
+                const results = await auth.listUsers("", after, MAX_LISTING_AMOUNT);
```

Comment:
Check out `useAPIWithPagination` I think it reduce a bit of code

Reply:
I don't think it's the same case.
Here, I need to load all the users, not just a specific page.
The useAPIWithPagination load each time a different page to the state which is not what I need